src/main/java/com/onarandombox/MultiverseCore/MultiverseCore.java

```diff
@@ -330,6 +330,34 @@ public String getAuthors() {
         return authors.toString();
     }
 
+    @Nullable
+    public File getBukkitConfig() {
```

Comment:
I'm not crazy about adding these methods to MultiverseCore after I worked to move most of our API stuff out of it. Is this functionality that is going to be useful elsewhere? Instead, we could inject the Server into the command class and just make these private methods in there.